application/init.php

```
@@ -26,23 +26,13 @@
 }
 
 // Sentry integration
-if (getenv('SENTRY_API_URL') || (defined('SENTRY_API_URL') && APPLICATION_ENV !== 'development')) {
+if (getenv('SENTRY_API_URL') || (defined('SENTRY_API_URL'))) {
     $sentryApiUrl = getenv('SENTRY_API_URL') ?: SENTRY_API_URL;
-    $ravenClient = new Raven_Client(
-        $sentryApiUrl,
-        array(
-            'environment' => APPLICATION_ENV,
-            'release' => (string)new Garp_Version,
-            'tags' => array(
-                'php_version' => phpversion(),
-            ),
-        )
-    );
-    $ravenErrorHandler = new Raven_ErrorHandler($ravenClient);
-    $ravenErrorHandler->registerExceptionHandler();
-    $ravenErrorHandler->registerErrorHandler();
-    $ravenErrorHandler->registerShutdownFunction();
-    Zend_Registry::set('RavenClient', $ravenClient);
+    \Sentry\init([
+        'dsn' => $sentryApiUrl,
+        'release' => strval(new Garp_Version),
+        'environment' => APPLICATION_ENV,
```

"Capture silenced errors" klikt als een nuttig instelling voor Garp: https://docs.sentry.io/platforms/php/#capture_silenced_errors 

Geen idee hoe het werkt maar het klinkt nuttig.